### ISSUE TITLE
"Flow as Tool" and "Sub Flow" - crashes in the same way

### ISSUE THREAD

**Issue by Gluepower** [langflow-ai/langflow#3183]
I am using Langflow v1.0.14 on Render. Vanilla instance, just standard flows. 

When trying to add component "Flow as Tool" it shows the list of flows that I have but when I try to select one it will just put a check box next to it in dropdown but will apply change. If I click to refresh flows to select again it will crash. 

![image](https://github.com/user-attachments/assets/b7b8eeea-aa86-4131-b961-66f06a36a01c)

After that it gives this message
![image](https://github.com/user-attachments/assets/3ab300f3-bed5-4c07-b13d-34683183b28f)

**Comment by lucaseduoli** [comment]
Hello! The issue has been resolved, but it wasnt released yet. In 1.0.15, your problem should be fixed.

Thanks! If the issue persists after 1.0.15, feel free to reopen this issue.

**Comment by Gluepower** [comment]
Hi @lucaseduoli I pulled and applied this fix: flow experimental components (https://github.com/langflow-ai/langflow/pull/3093)
The error was different before, didn't crash so I reported this after I applied that.  I might be doing something wrong as I am not a coder and first time doing this on render. Is that the right fix? To reproduce issue I use two basic hello world examples and just add add flow component to ensure it is not something unique to my setup. 
![image](https://github.com/user-attachments/assets/06045ab0-9d5e-43ff-8e69-2b7a1e993f82)
